web/pgadmin/static/js/Theme/overrides/szhmenu.override.js

```diff
@@ -23,12 +23,26 @@ export default function szhMenuOverride(theme) {
     '& .szh-menu__item': {
       display: 'flex',
       padding: '3px 12px',
+      // Add shortcut key style.
+      '& .menu-shortcut-key': {
```

Comment:
we use MUI for styling. Don't use classes like this. This file is for overriding. szhmenu